tests/api/conftest.py

```diff
@@ -12,17 +14,64 @@ def add_and_delete_new_pet():
     required_payload = {
         "name": fake.name(),
         "id": fake.pyint(),
-        "photoUrls": [
-            fake.url()
-            for _ in range(3)  # range now contents 3 urls
-        ],
+        "photoUrls": [fake.url() for _ in range(3)],  # range now contents 3 urls
     }
-    response = requests.post(f"{URL}pet",
-                             headers={"Accept": "application/json"},
-                             json=required_payload)
+    response = requests.post(
+        f"{URL}/pet", headers={"Accept": "application/json"}, json=required_payload
+    )
     yield response
     requests.delete(f"{URL}pet/{response.json().get('id')}")
 
 
+@pytest.fixture
+def login_user():
+    requests.get(
+        f"{URL}/user/login",
+        params={"username": "john", "password": "12345"},
```

Comment:
please store credentials in a separate file. [Beter an ini ](https://docs.pytest.org/en/stable/reference/customize.html#configuration-file-formats)

What if the user will be revoked? Ofc it would fail :)